TOC.md

```diff
@@ -381,6 +381,7 @@
     - [4.0.0-beta.1](/releases/4.0.0-beta.1.md)
     - [4.0.0-beta](/releases/4.0.0-beta.md)
   + v3.1
+    - [3.1.0-ga](/releases/3.1.0-ga.md)
```

Comment:
@TomShawn 3.0 GA uses upper-case `GA`. What do you suggest?

Reply:
Yes, you are right.